views/apartments/index.erb

```diff
@@ -0,0 +1,33 @@
+<script type="text/javascript">
+  $(document).ready(function(){
+      $("button").click(function(){
+        if (this.innerHTML == "Add An Apartment"){
+          $("form").hide(1000);
+          $("#create").toggle('slow','swing');
+        }
+      });
+  });
+
+</script>
+
+<!--Buttons for selection options  -->
+<button type="button">Add An Apartment</button>
+
+
+<!--Create Form Apartment-->
+<form id = "create" action = '/apartments/new' method="post" style="display:none; width:15vw;">
+  <input name='apartmentAddress' placeholder="Add Apartment Address">
+  <input name='apartmentMonthlyRent' placeholder="Add Apartment Rent (optional)">
```

Comment:
You can nest the names on forms, which will clean up the params that are submitted when this form gets submitted, and in turn cleans up the controller actions. Additionally, it's better to also include labels instead of just relying on placeholder text.
```html
  <label for="apartment[sqft]">sqft</label>
  <input type='text' name='apartment[sqft]' placeholder="2000">
```